### PR TITLE
fix: update bert-finetune-pytorch

### PR DESCRIPTION
**Pull request by sagnik**
related to #581

### CODE REVIEW

**Comment by dpressel** [comment]
Lets rename it to sst2-bert-base-uncased-hf (for HuggingFace)?

**Comment by sagnik** [comment]
renamed to `sst2-bert-hf-base-uncased.json` to be comparable with the tf one `sst2-bert-official-base-uncased.json`

File: mead/config/sst2-bert-hf-base-uncased.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "model_type": "fine-tune"
   },
   "train": {
-    "epochs": 5,
+    "epochs": 1,
```
Comment (dpressel) [inline mead/config/sst2-bert-hf-base-uncased.json:30]: Did you mean to change number of epochs?  If you want to test this, you can pass

`mead-train --x:train.epochs 1 ...`

Comment (sagnik) [reply mead/config/sst2-bert-hf-base-uncased.json:30]: oh i didnt.

Comment (sagnik) [reply mead/config/sst2-bert-hf-base-uncased.json]: updated

**Pull request merged**